pkg/utils/downloaders.go

```diff
@@ -129,6 +129,9 @@ func genEVELiveImage(image, outputDir string, format string, configDir string, s
 	if format == "gcp" {
 		fileName = fileName + ".img.tar.gz"
 	}
+	if format == "parallels" {
+		fileName = fileName + ".parallels"
```

Comment:
What about additional file alongside the image we need to use for parallels to run?